### PR TITLE
#3938 patient history

### LINKED ISSUE #3938: show patient history in vaccine dispensing
## Is your feature request related to a problem? Please describe.

- Want to show the patients history in vaccine dispensing

## Describe the solution you'd like

Not sure, will figure it out!

## Implementation

TBD

## Describe alternatives you've considered

**A clear and concise description of any alternative solutions or features you've considered.**

## Additional context

**Add any other context or screenshots about the feature request here.**

### PR DESCRIPTION
**Pull request by joshxg**
Fixes #3938 

## Change summary

- Adds a button on the last page (I didn't know where to put it. There's not a nice place on step two..)

## Testing

- [ ] Can open a modal with the patients vaccine history from the history button on step 3 of vaccine dispensing

### Related areas to think about

N/A

### CODE REVIEW

**Comment by joshxg** [comment]
image
image

**Review by mark-prins — APPROVED** [review]
that all looks awesome! works nicely for me

File: src/hooks/useSortableTable.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import { useState } from 'react';
+
+export const useSortableTable = (initialSortKey, initialIsAscending = true) => {
```
[the remark this answers is not in the document]
Comment (joshxg) [reply src/hooks/useSortableTable.js]: I think it matters :)

And I think usually a person who hasn't written the code generally has better names for stuff!

0c7fa82

File: src/selectors/Entities/name.js [hunk 2]
```diff
@@ -75,3 +75,15 @@ export const selectCanEditPatient = state => {
 
   return UIDatabase.getPreference(PREFERENCE_KEYS.CAN_EDIT_PATIENTS_FROM_ANY_STORE) || isEditable;
 };
+
+export const selectVaccinePatientHistoryHistory = state => {
```
[the remark this answers is not in the document]
Comment (joshxg) [reply src/selectors/Entities/name.js]: damn copy paste. It was meant to be `selectVaccinePatientPatientHistoryHistoryHistory`!!


0c23ec5

**Pull request merged**